scanpy/plotting/_tools/__init__.py

```diff
@@ -96,6 +96,26 @@ def pca_loadings(
         If `True` or a `str`, save the figure.
         A string is appended to the default filename.
         Infer the filetype if ending on {`'.pdf'`, `'.png'`, `'.svg'`}.
+
+    Examples
+    --------
+    .. plot::
+        :context: close-figs
+
+        import scanpy as sc
+        import numpy as np
+        import pandas as pd
+        adata = sc.datasets.pbmc68k_reduced()
+        sc.tl.pca(adata, svd_solver='arpack')
```

Comment:
I'm not sure we actually need to show the PCA being computed. Plus it takes up CI time and it'd be nice to keep these minimal.

Reply:
hey @ivirshup thanks,
the precomputed sc.datasets.pbmc68k_reduced throws an error so the PCA has to be computed for this to work (see #1816 ). i didn't think it was a big issue to recalc, I do get your point on showing the step without calculating if it's not necessary. 
how do you suggest to solve this? what is a better adata to start from, i guess with pbmc3k_processed the processing would be the same?

sorry for the h5ad will remove and update the commit when you let me know!

Reply:
Yeah, I'd probably just use `pbmc3k_processed` for this. If you want to show the `pca` call, but not have it run, you can do something like:

```rst
.. code:: python

    import scanpy as sc
    
    pbmc = sc.datasets.pbmc3k_processed()
    sc.pp.pca(pbmc)
```

You could probably even hide the `import` and the data loading if you wanted.

> i didn't think it was a big issue to recalc

It's not, especially for PCA. It becomes more of an issue for things like `umap` or `tsne`. I figure it's better to just be consistent about it, so someone could pretty much copy what's done here for those functions.